### DISCUSSION TITLE
Caching QC calculations between sessions in BespokeFit

### DISCUSSION THREAD

**Discussion by orionarcher** [openforcefield/discussions#28] [Infrastructure]
I am interested in using BespokeFit on a large number of molecules (~50-200) with similar chemistries. I would like to be able to run some molecules initially, and then add more over time. It seems clear that many QC parameterization calculations could be reused since the chemistries are similar. However, I can't find a way to instruct BespokeFit to draw QC calculations from an existing database/cache, if they are available. I have two questions:

1. is it possible to reuse QC calculations between multiple molecules with BespokeFit?

2. is it possible to cache those QC results for reuse in BespokeFit at a later time?

Any advice appreciated!

**Comment by davidlmobley** [comment]
This should involve interfacing it with a local instance of QCArchive, which is possible and, indeed, BespokeFit was designed to make that possible. I haven't done this myself, though, but I will point a couple relevant folks to this in case they have time to give pointers. (Our main BespokeFIt documentation guy is temporarily diverted to other projects so he can't get to this particularly soon.)

But yes, the high level answer to both of your questions is "yes, using a local QCArchive instance".

**Comment by orionarcher** [comment]
Great to know, thanks for the response. Hope to see some example of this at some point!
